prometheus_client/openmetrics/parser.py

```diff
@@ -364,6 +364,101 @@ def _parse_remaining_text(text):
     return val, ts, exemplar
 
 
+def _parse_nh_sample(text, suffixes):
+    labels_start = text.find("{")
+    # check if it's a native histogram with labels
+    re_nh_without_labels = re.compile(r'^[^{} ]+ {[^{}]+}$')
+    re_nh_with_labels = re.compile(r'[^{} ]+{[^{}]+} {[^{}]+}$')
+    print('we are matching \'{}\''.format(text))
```

Comment:
We should remove the debug printing before merging, there are a couple other lines in this function as well.

Reply:
I could see just another one, I hope I removed them all now XD